### ISSUE TITLE
Mime::Type::InvalidMimeType from append_info_to_payload

### ISSUE THREAD

**Issue by kaoru** [skylightio/skylight-ruby#150]
In lib/skylight/core/probes/action_controller.rb it looks like skylight wraps the `append_info_to_payload` method, calling the original and then appending two additional keys `:sk_rendered_format` and `:sk_variant`.

I'm getting some exceptions for some requests. `sk_rendered_mime` is throwing Mime::Type::InvalidMimeType

```
"#<Mime::NullType:0x000055bc05e5c678>" is not a valid MIME type


lib/action_dispatch/http/mime_type.rb:143 new
lib/action_dispatch/http/mime_type.rb:143 lookup
skylight-core (4.3.0) lib/skylight/core/probes/action_controller.rb:19 sk_rendered_mime
skylight-core (4.3.0) lib/skylight/core/probes/action_controller.rb:12 append_info_to_payload
lib/active_record/railties/controller_runtime.rb:44 append_info_to_payload
app/controllers/application_controller.rb:237 append_info_to_payload
```

The requests that are causing this seem to be malicious, for example `GET /wp-login.php`. But I still don't think the skylight gem should be throwing exceptions even on the weirdest requests.

**Comment by zvkemp** [comment]
Hi @kaoru, I was able to reproduce this by piping an unrestricted catch-all route through an ActionController instance. I think the root cause is probably something in actionpack or mime-types (it's returning `"#<Mime::NullType:0x000055bc05e5c678>"` as a string, which doesn't really make sense), but I agree that we should avoid surfacing this error. We will investigate further.

**Comment by kaoru** [comment]
Nice! Thanks @zvkemp

**Comment by zvkemp** [comment]
@kaoru I just released version [4.3.1](https://rubygems.org/gems/skylight/versions/4.3.1) (commit https://github.com/skylightio/skylight-ruby/commit/47f370c55258c4972d7bccded2d3cc220c395179) which should fix your issue.

**Comment by zvkemp** [comment]
As an aside, the root cause of the issue appears to have been corrected in https://github.com/rails/rails/pull/37617, which will be released with Rails 6.1.
